typescript-react/README.md

```diff
@@ -30,6 +30,11 @@ An [example](./src/containers/Listings/Listings.tsx) of a card is provided along
 This sub-page should load and display a simple list of the price history for a specific listing.
 The sub-page should be accessible through the URL `/:listingId/prices`.
 
+## Objective 4: Answer the questions in the pull request
+
+When you will create the pull request to hand over your work, it will contain a few questions. Do not forget to plan for some time to answer all of them. You can 
+read more about it in the [main README file](../README.md).
```

Comment:
Maybe linking directly to the apprioriate section like the following could be interesting ? 
```suggestion
read more about it in the [main README file](../README.md#iii---handing-over-your-work).
```

Reply:
Great idea, thanks!